app/test/unit/parse-app-url-test.ts

```diff
@@ -24,7 +24,7 @@ describe('parseAppURL', () => {
     })
   })
 
-  describe('openRepo', () => {
+  describe('openRepoViaHttps', () => {
```

Comment:
:art: these don't need to be squished together like that (it was just that way because `openRepo` is the command. 

What about if we do `openRepo via HTTPS` and `openRepo via SSH below`?

Reply:
as comments or seperate `describe`s

Reply:
Just use spaces in the first parameter string so when they displayed in the test runner they're human-friendly text.